zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/BrokerAdminServiceEndpointTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.it.management;
+
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+
+import io.camunda.zeebe.qa.util.cluster.TestStandaloneBroker;
+import io.camunda.zeebe.qa.util.cluster.TestZeebePort;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration;
+import io.camunda.zeebe.qa.util.junit.ZeebeIntegration.TestZeebe;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.filter.log.RequestLoggingFilter;
+import io.restassured.filter.log.ResponseLoggingFilter;
+import io.restassured.http.ContentType;
+import io.restassured.specification.RequestSpecification;
+import java.util.concurrent.TimeUnit;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+@ZeebeIntegration
+public class BrokerAdminServiceEndpointTest {
+
+  static RequestSpecification brokerServerSpec;
+
+  @TestZeebe(initMethod = "initTestStandaloneBroker")
+  private static TestStandaloneBroker broker;
+
+  @SuppressWarnings("unused")
+  static void initTestStandaloneBroker() {
+    broker = new TestStandaloneBroker().withProperty("management.server.base-path", "/foo");
+  }
+
+  @BeforeAll
+  static void setUpClass() {
+    brokerServerSpec =
+        new RequestSpecBuilder()
+            .setContentType(ContentType.JSON)
+            // set URL explicitly since we want to ensure the mapping is correct
+            .setBaseUri("http://localhost:" + broker.mappedPort(TestZeebePort.MONITORING) + "/foo")
+            .addFilter(new ResponseLoggingFilter())
+            .addFilter(new RequestLoggingFilter())
+            .build();
+  }
+
+  @Test
+  void shouldReturnPartitions() {
```

Comment:
:wrench: Could we test an expected response? Just a basic response I think is fine. That way if we also change the output, we would notice it could be breaking.

Reply:
I added an expected json for both routes. Unfortunately there is a timestamp in the response, so I had to replace that timestamp to a "static" value before checking.